Task3/Task3/Task3/Figures/FilmCircle.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Task3.Exceptions;
+using Task3.Sheets;
+
+namespace Task3.Figures
+{
+    public class FilmCircle : Figure, IFilm
+    {
+        public FilmCircle()
+        {
+
+        }
+
+        public FilmCircle(double radius)
+        {
+            Side = radius;
+        }
+
+        public FilmCircle(ref FilmEquilateralTriangle triangle, double side)
+        {
+            if (side <= triangle.Side / (2 * Math.Sqrt(3)))
```

Comment:
Can we move this logic to another private method?

Reply:
fixed